src/compose/network.ts

```diff
@@ -217,7 +217,9 @@ export class Network {
 	}
 
 	private static validateComposeConfig(
-		config: Partial<ComposeNetworkConfig>,
+		config: Partial<Omit<ComposeNetworkConfig, 'ipam'>> & {
+			ipam?: Partial<ComposeNetworkConfig['ipam']>;
+		},
```

Comment:
Updated typescript picked up that this is the actual type we call this with so I had to amend it here to accept it